src/hardware/imfc_rom.c

```diff
@@ -0,0 +1,1061 @@
+
```

Comment:
Why do we need this file at all? It is not included in the compilation, `imfc.cpp` seems to contain the data too.

BTW, moving the binary data out of `imfc.cpp` is, IMHO, a good idea.

Reply:
Line 23 claims it features memory tables for the IBM Music Feature Card,

Reply:
> BTW, moving the binary data out of imfc.cpp is, IMHO, a good idea.

Yes; it makes it a lot easier to read through the code. The `.c` file is  `#include`d just above the `class MusicFeatureCard` where they're used.